tools/editor/editor_data.cpp

```diff
@@ -611,11 +611,16 @@ void EditorData::set_edited_scene(int p_idx){
 	current_edited_scene=p_idx;
 	//swap
 }
-Node* EditorData::get_edited_scene_root(){
+Node* EditorData::get_edited_scene_root(int idx){
 
 	ERR_FAIL_INDEX_V(current_edited_scene,edited_scene.size(),NULL);
```

Comment:
I get this should be in the first `if` block (same for the other methods with an added index).